### DISCUSSION TITLE
Some script can be run on tampermonkey beta, but not on violentmonkey

### DISCUSSION THREAD

**Discussion by yeezylife** [violentmonkey/violentmonkey#2155] [Ideas]
for example this one here https://github.com/syhyz1990/baiduyun

what's the problem and will violentmonkey support these kind of scripts later?

**Comment by yeezylife** [comment]
seems to be something wrong with the script's authorization process on violentmonkey

**Comment by tophf** [comment]
It'll run when GM_cookie is implemented, see #603.
